presto-druid/src/main/java/com/facebook/presto/druid/DruidQueryGeneratorContext.java

```diff
@@ -244,14 +244,14 @@ private boolean hasAggregation()
         }
 
         String expressions = selections.entrySet().stream()
-                .map(s -> s.getValue().getDefinition())
+                .map(s -> s.getValue().getEscapedDefinition())
                 .collect(Collectors.joining(", "));
         if (expressions.isEmpty()) {
             throw new PrestoException(DRUID_QUERY_GENERATOR_FAILURE, "Empty Druid query");
         }
 
         String tableName = from.orElseThrow(() -> new PrestoException(DRUID_QUERY_GENERATOR_FAILURE, "Table name missing in Druid query"));
-        String query = "SELECT " + expressions + " FROM " + tableName;
+        String query = "SELECT " + expressions + " FROM \"" + tableName + "\"";
```

Comment:
private function to add quotes to table or column names. could be used here

Reply:
done